### DISCUSSION TITLE
User trouble in sonarr/radarr/sabnzbd

### DISCUSSION THREAD

**Discussion by ApriliaEdd** [tteck/Proxmox#985] [Help Needed]
Hi, 
Great work on these scripts they are brilliant.  I have recently used the sabnzbd, sonarr and radarr scripts and noticed a couple of issues, one of which I saw you fixed earlier regarding the version of unrar used in sabnzbd.  Good job fixing this I may redeploy mine as I'm sure my many different tries to get this to work wont have helped as my linux skills are terrible.
The other issue I noticed was to do with the users that are used to install the applications within the scripts.  I have a share mounted from my unraid server to give sonar/radarr/sabnzbd access to my media library when I edited the fstab file in the sonarr lxc it couldn't write to the share.  I found that it was using a user called sonarr.  once I change the uid in the fstab file it all worked well.  I was then expecting similar behaviour from the radarr one but this worked using my uid.
I prefer not to have root ssh enabled if possible.  I create my own admin user for ssh access and copy ssh keys over after the container is built.
I found a similar issue with node red when I wanted to copy and paste the contents of my packages.json file but as it was installed using root I could not do it over ssh (i didnt have root ssh enabled).
Would it be possible to have the option under advanced setup to add a user and password, make it sudo and then install the application using it?  This way when I remote in via ssh using my admin user I have access to all the install locations.

thanks again for all your work.

**Comment by AAH134** [comment]
Hey I am facing the same issue, radarr working fine but sonar keep giving me error cannot write to file. 
how did you change the uid? is it don in the container or the proxmox?

**Reply by ApriliaEdd** [reply]
I chose not to change the UID as I assumed all the file ownerships would need changing then also which is way above my linux level.
In the container console view the contents of /etc/passwd you will find the UID and GID for the sonarr user there.  You can then change your fstab entry to match the UID & GID from /etc/passwd thus using the sonarr user to access your share.
